src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -557,46 +555,42 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
+register_cli_argument('network dns record-set cname set-record', 'record_set_name', options_list=('--record-set-name', '-n'), help='The name of the record set relative to the zone. Creates a new record set if one does not exist.')
+
+register_cli_argument('network dns record-set soa', 'relative_record_set_name', ignore_type, default='@')
+
+register_cli_argument('network dns record-set a', 'ipv4_address', options_list=('--ipv4-address', '-a'), help='IPV4 address in string notation.')
+register_cli_argument('network dns record-set aaaa', 'ipv6_address', options_list=('--ipv6-address', '-a'), help='IPV6 address in string notation.')
+register_cli_argument('network dns record-set cname', 'cname', options_list=('--cname', '-c'), help='Canonical name.')
+register_cli_argument('network dns record-set mx', 'exchange', options_list=('--exchange', '-e'), help='Exchange metric.')
+register_cli_argument('network dns record-set mx', 'preference', options_list=('--preference', '-p'), help='Preference metric.')
+register_cli_argument('network dns record-set ns', 'dname', options_list=('--nsdname', '-d'), help='Name server domain name.')
+register_cli_argument('network dns record-set ptr', 'dname', options_list=('--ptrdname', '-d'), help='PTR target domain name.')
+register_cli_argument('network dns record-set soa', 'host', options_list=('--host', '-t'), help='Host name.')
+register_cli_argument('network dns record-set soa', 'email', options_list=('--email', '-e'), help='Email address.')
+register_cli_argument('network dns record-set soa', 'expire_time', options_list=('--expire-time', '-x'), help='Expire time (seconds).')
+register_cli_argument('network dns record-set soa', 'minimum_ttl', options_list=('--minimum-ttl', '-m'), help='Minimum TTL (time-to-live, seconds).')
+register_cli_argument('network dns record-set soa', 'refresh_time', options_list=('--refresh-time', '-f'), help='Refresh value (seconds).')
+register_cli_argument('network dns record-set soa', 'retry_time', options_list=('--retry-time', '-r'), help='Retry time (seconds).')
+register_cli_argument('network dns record-set soa', 'serial_number', options_list=('--serial-number', '-s'), help='Serial number.')
+register_cli_argument('network dns record-set srv', 'priority', options_list=('--priority', '-p'), help='Priority metric.')
+register_cli_argument('network dns record-set srv', 'weight', options_list=('--weight', '-w'), help='Weight metric.')
+register_cli_argument('network dns record-set srv', 'port', options_list=('--port', '-r'), help='Service port.')
+register_cli_argument('network dns record-set srv', 'target', options_list=('--target', '-t'), help='Target domain name.')
+register_cli_argument('network dns record-set txt', 'value', options_list=('--value', '-v'), nargs='+', help='Space separated list of text values which will be concatenated together.')
```

Comment:
Will it be concatenated to include the space or not?
Why `nargs='+'` if I can just pass in a single string that has my own spaces in?